TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -205,7 +205,7 @@ public int GetTestCount()
                     case "ExecSqlInDataDivision.rdz.cbl":
                     case "Program.pgm":        
                     case "ParagraphSection.rdz.cbl":
-                    case "ExecSqlWithCommit.rdz.cbl":
+                    //case "ExecSqlWithCommit.rdz.cbl":
```

Comment:
No need to comment code, remove it. Commented code brings confusion for readers, we'll alaways have git history to see previous modifications